### PR TITLE
Added allowNewEnroll on SmartSelfie, Biometric, DocV and EnhancedDocV

### PR DESCRIPTION
**Pull request by jumaallan**
Story: https://app.shortcut.com/smileid/story/xxx

## Summary

Added allowNewEnroll on SmartSelfie, Biometric, DocV and EnhancedDocV

## Known Issues

Any shortcomings in your work. This may include corner cases not correctly handled or issues related
to but not within the scope of your PR. Design compromises should be discussed here if they were not
already discussed above.

## Test Instructions

Concise test instructions on how to verify that your feature works as intended. This should include
changes to the development environment (if applicable) and all commands needed to run your work.

## Screenshot

If applicable (e.g. UI changes), add screenshots to help explain your work.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+## 10.0.3
```
Comment (mldangelo) [inline CHANGELOG.md:5]:
```suggestion
```

**Review by mldangelo — APPROVED** [review]
Looks good! Please review my nit on the changelog. Thanks @jumaallan

File: CHANGELOG.md (hunk 1 above)
Comment (jumaallan) [reply CHANGELOG.md:5]: We usually create the next release as an empty block before doing a release

Comment (mldangelo) [reply CHANGELOG.md:5]: Ah. How do you know that the next release will be a patch? https://keepachangelog.com/en/1.1.0/ Recommends an unreleased block at the top to track changes for the next release.

**Review by mldangelo — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (jumaallan) [reply CHANGELOG.md:5]: We leave these two at the top

```
## Unreleased

## 10.0.3

````

Comment (jumaallan) [reply CHANGELOG.md:5]: If it is not a patch, someone ammends it and bumps again and leaves an empty block

Comment (mldangelo) [reply CHANGELOG.md:5]: In my opinion, we should not anticipate the next version number until we are ready to do a release. This is not aligned with what most other people do, but it’s ok if our conventions are different. Feel free to ship it!

File: lib/src/main/java/com/smileidentity/compose/SmileIDExt.kt [hunk 2]
```diff
@@ -114,6 +117,7 @@ fun SmileID.SmartSelfieAuthentication(
     userId: String,
     modifier: Modifier = Modifier,
     jobId: String = rememberSaveable { randomJobId() },
+    allowNewEnroll: Boolean = false,
```
Comment (vanshg) [inline lib/src/main/java/com/smileidentity/compose/SmileIDExt.kt:120]: @jumaallan Can you add KDocs to all composables where the new parameter was added?
